traveler-swift-core/TravelerKit/Traveler.swift

```diff
@@ -589,7 +589,7 @@ public class Traveler {
         A `CancellationError.expiredQuote` will be thrown if the quote has expired.
      */
 
-    public static func cancelOrder(quote: CancellationQuote, completion: @escaping (Error?) -> Void) {
+    public static func cancelOrder(quote: CancellationQuote, completion: @escaping (Order?,Error?) -> Void) {
```

Comment:
Swift convention! ^

white spacing. 

I will eventually add a lynt that will automatically reject any PRs that dont follow the conventional white spacing and format.